styles/likely.styl

```diff
@@ -160,6 +160,7 @@ colorize('gplus',         rgb(221,66,65)); // #dd4241
 colorize('vkontakte',     rgb(82,110,143)); // #526e8f);
 colorize('pinterest',     rgb(204,0,43)) // #cc002b
 colorize('odnoklassniki', #f6903b);
+colorize('telegram', #34ABDE)
```

Comment:
Also, `#34ABDE` should be in the lower case and aligned with other color values, such as:

```js
colorize('odnoklassniki', #f6903b);  
colorize('telegram',      #34abde) ;
```